src/Sulu/Bundle/AdminBundle/Navigation/ContentNavigationItem.php

```diff
@@ -241,7 +241,7 @@ public function toArray()
             'action' => $this->getAction(),
             'display' => $this->getDisplay(),
             'component' => $this->getComponent(),
-            'componentOptions' => $this->getComponentOptions(),
+            'componentOptions' => array_merge($this->getComponentOptions(), $options),
```

Comment:
Not related, I guess, to this PR - but would it make sense to stop using `->toArray` generally and use the JMS serializer? /cc @wachterjohannes